ouroboros-consensus/src/Ouroboros/Consensus/Mempool/API.hs

```diff
@@ -100,11 +100,13 @@ data Mempool m blk idx = Mempool {
       -- index of transaction hashes that have been included on the blockchain.)
       addTxs :: [GenTx blk] -> m [(GenTx blk, ApplyTxErr blk)]
 
-      -- | Get all transactions in the mempool (oldest to newest)
+      -- | Get all transactions in the mempool along with their associated
+      -- ticket numbers (oldest to newest).
       --
-      -- Guarantees that these transactions will be valid (when applied strictly
-      -- in order) with respect to a call to 'getLedgerState' run /in the same
-      -- transaction/.
+      -- n.b. This function provides /no guarantee/ that the resulting
+      -- transactions will be valid. However, it is guaranteed that these
```

Comment:
valid with respect to the current ledger state of the chain db.

We should document somewhere in this API (perhaps right here) that `getTxs` is eventually consistent, in the sense that it eventually (async) picks up changes in the ledger state, but it is not syncronous (without using syncState).